### PR TITLE
Add "npm run dbtots" to generate ts files from planetscale db

### PR DESCRIPTION
**Pull request by seanim0920**
Added packages and scripts to generate typescript types from our planetscale schema and save them in the TiFBackendUtils package. This removes the need for us to manually write types and interfaces matching the planetscale db and keep them in sync.
We just need to remember to run "npm run dbtots" frequently.

Note: Requires .env in tifbackendutils folder containing the database host, username and password.
Do not edit the autogenerated file. If new types are needed, add them to a separate file.

### CODE REVIEW

**Review by TCbacon — DISMISSED** [review]
Auto generating the entities will help with having to do it manually. Currently, will still need to manually create custom types (such as blocked types), unions, and intersections.

**Review by suryatiru — DISMISSED** [review]

File: APILambda/package.json [hunk 1]
```diff
@@ -9,6 +9,7 @@
   "description": "",
   "main": "index.js",
   "scripts": {
+    "dbtots": "node ../TiFBackendUtils/Planetscale/dbToTs.mjs",
```
Comment (seanim0920) [inline APILambda/package.json:12]: Try a pre-commit hook in case we forget to run this

File: TiFBackendUtils/Planetscale/entities.ts [hunk 2]
```diff
@@ -0,0 +1,333 @@
+/* eslint-disable @typescript-eslint/ban-types */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+/* eslint-disable quotes */
+/*
+* This file was generated by a tool.
+* Rerun sql-ts to regenerate this file.
+*/
+export interface DBbanned {
```
Comment (seanim0920) [inline TiFBackendUtils/Planetscale/entities.ts:8]: Add these to the README. Remember to remove the planetscale stuff somehow

File: .github/workflows/test_api.yml [hunk 3]
```diff
@@ -41,6 +41,7 @@ jobs:
         run: |
           cd TiFBackendUtils
           npm ci
+          npm run dbtots
```
Comment (seanim0920) [inline .github/workflows/test_api.yml:44]: Remove after pre-commit hook

**Pull request merged**